src/ontology/mondo-ingest.Makefile

```diff
@@ -169,6 +169,28 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+# TODO: Add .PHONY here and below to top of file?
+# TODO: need a purl for this?
+# TODO: use this target: tmp/mondo.sssom.tsv
+.PHONY: mondo-mappings
+MONDO_MAPPINGS_TSV = 'https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv'
+mondo-mappings:
+	wget ${MONDO_MAPPINGS_TSV} -P ../analysis/excluded_terms_in_mondo/ignored/mappings/
+
+# TODO: Need to add dependency for sig/mirror files?
+.PHONY: excluded_terms_in_mondo
```

Comment:
This does not seem to be used?

Reply:
just haven't finished makefile edits yet but yes this will be removed

Reply:
done